runelite-client/src/main/resources/net/runelite/client/plugins/skybox/skybox.txt

```diff
@@ -955,6 +955,13 @@ c 6 3
 c 7 4
 
 
+m31 89
+
+// 31 89 p1 Christmas-themed POH
+p 1
+#B6CBD8
+C 0 0 7 15
+
```

Comment:
you are missing dungeon colors still

Reply:
Oops. Do you think white would suit the dungeon, considering dungeons are underground? I was thinking either `#484840` or `#2c2c29`.